test-end-to-end/webapp-cypress/cypress/e2e/007_005_student_registered_by_admin_or_teacher_dont_need_parentConsent.cy.js

```diff
@@ -0,0 +1,26 @@
+describe('007_004_user_registration_dob_field_is_mandatory', () => {
+ it('Start Ustad Test Server ', () => {
+  // Start Test Server
+    cy.ustadStartTestServer()
+})
+
+it('User registration date of birth field is mandatory', () => {
+ // Admin user login
+  cy.ustadClearDbAndLogin('admin','testpass',{timeout:8000})
+  cy.contains("People").click()
+  cy.contains("button","Person").click()
+  cy.contains("label", "First names").parent().find("input").clear().type("Student")
+  cy.contains("label", "Last name").parent().find("input").clear().type("A")
+  cy.get('div[id="gender"]').click()
+  cy.contains("li","Female").click()
+  cy.ustadBirthDate(cy.get("#person_date_of_birth"), new Date("2017-06-01"))
```

Comment:
Can this be done based on current system time?

e.g. timeNow - (365 x 24 x 60 x 60 x 1000) (365 days x 24 hours x 60 min x 60 sec x 1000 millsecs). That is not exactly five years ago, but its close enough. If we still have this test in 2030, it would stop for the intended purpose.